packages/credentials-interface/package.json

```diff
@@ -47,7 +47,7 @@
     "url": "git+https://github.com/energywebfoundation/ew-did-registry.git"
   },
   "dependencies": {
-    "@sphereon/pex": "^1.1.0",
+    "@sphereon/pex": "1.1.3",
```

Comment:
Is this change relevant to this PR? It's a different package, so I think it should be removed

Reply:
`@sphereon/pex` introduced breaking changes in 1.1.4

Reply:
Ok, thanks for the explanation. This maybe indeed be a good change to make. Still, I think it is separate to this PR, so would be best made in a separate PR, so that we can more easily revert/isolate the change if necessary. What do you think?

Reply:
Ok, make sence https://github.com/energywebfoundation/ew-did-registry/pull/460